ReportSimulationOutput/measure.rb

```diff
@@ -1189,6 +1205,58 @@ def sanitize_name(name)
       end
     end
 
+    # Zone detailed conditions
+    if args[:include_timeseries_zone_conditions]
+
+      # Get zones, excluding zones used for modeling workarounds
+      zone_names = []
+      @model.getThermalZones.each do |zone|
+        next if zone.floorArea <= 1
+
+        zone_names << zone.name.to_s.upcase
+      end
```

Comment:
Looks like this is copied from the zone temperatures code. Can we combine the two so the logic only appears once? Could just calculate it above either one of these code blocks or could use a method.